src/services/big-query-client.ts

```diff
@@ -0,0 +1,41 @@
+import { ResultWithoutExpectedErrors } from '../doppler-types';
+import { ResponseCache } from './../utils';
+import { GenericClient } from './generic-client';
+
+export interface EmailList {
+  emails: String[];
+}
+
+export interface BigQueryClient {
+  getEmailsData(idUser: String): Promise<ResultWithoutExpectedErrors<EmailList>>;
+}
+
+export class HttpBigQueryClient extends GenericClient implements BigQueryClient {
+  private responseCache = new ResponseCache();
+
+  private mapEmails(response: any): EmailList {
+    return {
+      emails: response.emails,
+    };
+  }
+  public async getEmailsData(idUser: String): Promise<ResultWithoutExpectedErrors<EmailList>> {
+    try {
+      const { jwtToken } = super.getApiConnectionData();
+      const response = await this.axios.request({
+        method: 'GET',
+        url: `/big-query/emails`,
```

Comment:
What do you think about a more descriptive name for this URL?

For example: 

```suggestion
        url: `/big-query/allowed-emails`,
```

Reply:
change accepted and applied